worker/codalabworker/local_run/local_run_manager.py

```diff
@@ -0,0 +1,239 @@
+import logging
+import os
+from subprocess import check_output
+import threading
+import time
+import socket
+
+from run_manager import BaseRunManager, RunState
+from local_run_state import LocalRunStateMachine, LocalRunStage
+from local_reader import LocalReader
+
+logger = logging.getLogger(__name__)
+
+class LocalRunManager(BaseRunManager):
+    """
+    LocalRunManager executes the runs locally, each one in its own Docker
+    container. It manages its cache of local Docker images and its own local
+    Docker network.
+    """
+    def __init__(self, worker, docker, image_manager, dependency_manager,
+            state_committer, bundles_dir, cpuset, gpuset, docker_network_prefix='codalab_worker_network'):
+        self._worker = worker
```

Comment:
why are some members prepended with `_` while others aren't?

Reply:
Artifact of something I was trying earlier dealing with access to these from a member class, will change back to consistent.